apps/guardian-ui/src/components/setup/screens/roleSelector/RoleSelector.tsx

```diff
@@ -104,11 +106,18 @@ export const RoleSelector = React.memo<Props>(function RoleSelector({
           width={['100%', 'auto']}
           leftIcon={<Icon as={ArrowRightIcon} />}
           isDisabled={!role}
-          onClick={handleNext}
+          onClick={() => {
+            role !== GuardianRole.Solo ? setIsModalOpen(true) : handleConfirm();
```

Comment:
Nit: The `!==` leads me to mentally do a double-negative to grok this line. Could we instead do this?

`role === GuardianRole.Solo ? handleConfirm() : setIsModalOpen(true);`

Reply:
done